### PR TITLE
Corona Test Config (EXPOSUREAPP-6047)

### PR DESCRIPTION
**Pull request by mtwalli**
- Map corona test parameters from AppConfig

#### TODO
- [x] Align default value

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/CoronaTestConfig.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    val coronaRapidAntigenTestParameters: CoronaRapidAntigenTestParametersContainer
+
+    interface Mapper : ConfigMapper<CoronaTestConfig>
+}
+
+data class CoronaRapidAntigenTestParametersContainer(
+    val hoursToDeemTestOutdated: Int = DEFAULT_HOURS
+) {
+    companion object {
+        const val DEFAULT_HOURS = 48 // TODO align this
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/CoronaTestConfig.kt:15]: @mlenkeit FYI ☝️

Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/CoronaTestConfig.kt]: Aligned

**Review by LukasLechnerDev — APPROVED** [review]
LGTM

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/CoronaTestConfig.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    val coronaRapidAntigenTestParameters: CoronaRapidAntigenTestParametersContainer
+
+    interface Mapper : ConfigMapper<CoronaTestConfig>
+}
+
+data class CoronaRapidAntigenTestParametersContainer(
+    val hoursToDeemTestOutdated: Int = DEFAULT_HOURS
+) {
+    companion object {
+        const val DEFAULT_HOURS = 48
```
Comment (jurajkusnier) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/CoronaTestConfig.kt:15]: `DEFAULT_HOURS` could be `private`, or is it intended to be used somewhere else?

**Review by jurajkusnier — APPROVED** [review]

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/CoronaTestConfig.kt (hunk 2 above)
Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/CoronaTestConfig.kt:15]: 👍🏼 , will do in another PR , I planed to used publicly, but then I refactored it

**Pull request merged**